drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/src/test/java/org/drools/workbench/screens/scenariosimulation/webapp/client/editor/ScenarioSimulationEditorKogitoRuntimeScreenTest.java

```diff
@@ -74,7 +69,7 @@ public void setup() {
     public void getPlaceRequest() {
         assertEquals(SCENARIO_SIMULATION_KOGITO_RUNTIME_SCREEN_DEFAULT_REQUEST, scenarioSimulationEditorKogitoRuntimeScreenSpy.getPlaceRequest());
     }
-
+/*
```

Comment:
Dead code?

Reply:
@danielezonca Temporary disabled tests. I'll reimplement them in the next PR